llm_demo/evaluation/evaluation.py

```diff
@@ -140,3 +145,23 @@ def evaluate_response_phase(eval_datas: List[EvalData]) -> evaluation_base.EvalR
         experiment=RESPONSE_EXPERIMENT_NAME,
     ).evaluate()
     return eval_result
+
+
+INSTRUCTION = """The Cymbal Air Customer Service Assistant helps customers of Cymbal Air with their travel needs.
```

Comment:
consider how to keep this consistent with the original

Reply:
The only way to extract this is either import the instruction from the lc orchestration file. Else, another way to extract it is using callbacks. Eventually when we use langgraph, this could be extracted via state. :)